File: Part4/Rust/Function_MissingParameter.rs
use std::io;

fn main()
{
    let mut user_input = String::new();
    
    println!( "Type 1 and press enter." );
    io::stdin().read_line( &mut user_input ).expect( "Failed to read line" );
    
    // Function requires type parameter, but we're not providing it
    let entered_integer = user_input.trim().parse().unwrap();
    
    println!( "The user entered the integer {}", entered_integer );
}

File: Part4/C#/Function_MissingParameter.cs
using System;

class Program
{
    static void Main()
    {
        string userInput;
        
        Console.WriteLine("Type 1 and press enter.");
        userInput = Console.ReadLine();
        
        // Function requires 1 parameter, but we're passing 0
        int enteredInteger = int.Parse();
        
        Console.WriteLine("The user entered the integer {0}", enteredInteger);
    }
}
